web/src/components/DLP/OverviewTab.vue

```diff
@@ -328,5 +332,11 @@ onMounted(() => {
     await store.fetchDandiset({ identifier, version });
   }, 5000);
 });
+onUnmounted(() => {
+  if (timer === undefined) {
+    throw Error('Invalid timer value');
+  }
```

Comment:
To the extent that we think it's actually an error condition if the timer is not set at this point, this is fine. If it ever happens that we want to cancel the interval timer for some other reason, then I think we would invert this conditional, remove the `throw` statement, and not worry about a non-existing timer.

Reply:
I'll also add, the `throw` statement has the advantage of causing a sentry error if it somehow ever did get called.